apps/intake/tests/utils/Paperwork.ts

```diff
@@ -183,6 +182,38 @@ export class Paperwork {
   async skipPrimaryCarePhysician(): Promise<void> {
     await this.CommonLocatorsHelper.clickContinue();
   }
+  async fillPrimaryCarePhysician(): Promise<{  
+  firstName: string;
+  lastName: string;
+  pcpAddress: string;
+  pcpName: string;
+  formattedPhoneNumber: string;}> {
+    const firstName = `First name ${this.getRandomString()}`;
```

Comment:
suggestion: Random data in the tests may lead to different bugs. If you create random data in one test thread and check with that class in another thread, it will fail because each thread will have different random data. And test in that case will be flaky. I've already faced these issues in our tests. Why not use constants? Resources should be deleted after tests so we will start with a clean setup every time.